controllers/cloud.redhat.com/providers/appinterface_objectstore_test.go

```diff
@@ -0,0 +1,152 @@
+package providers
+
+import (
+	"fmt"
+	"testing"
+
+	"cloud.redhat.com/clowder/v2/controllers/cloud.redhat.com/config"
+	"cloud.redhat.com/clowder/v2/controllers/cloud.redhat.com/utils"
+	core "k8s.io/api/core/v1"
+)
+
+type TestSecrets struct {
+	ExactKeys   map[string]string
+	ExtraKeys   map[string]string
```

Comment:
These three (`ExtraKeys`, `NoKeys`, `MissingKeys`) are there to be able to create extra tests from?

Reply:
I did not finish my unit tests, but I ran out of time at the end of the day and figured I could come back later and add them.